itest/opstackl2/op_test_manager.go

```diff
@@ -109,18 +124,58 @@ func StartOpL2ConsumerManager(t *testing.T, numOfConsumerFPs uint8) *OpL2Consume
 	require.NoError(t, err)
 	t.Logf(log.Prefix("Register consumer %s to Babylon"), opConsumerId)
 
+	// define finality gadget configs
+	cfg := fgcfg.Config{
+		L2RPCHost:         opL2ConsumerConfig.OPStackL2RPCAddress,
+		BitcoinRPCHost:    "rpc.ankr.com/btc",
```

Comment:
iirc, this is not used. if that's the case, let's use a mock URL to avoid confusion?

Reply:
why is it not used? i think we still use it to query bitcoin block timestamps in `finality-gadget`

Reply:
oh so does that mean our e2e test depend on this ankr RPC? maybe I got confused with what we have in the OP unit tests where we used mock url